packages/utils/src/crypto/packet/padding.ts

```diff
@@ -0,0 +1,34 @@
+import { PAYLOAD_SIZE } from './constants'
+import { CryptoError } from '../cryptoError'
+import { u8aEquals } from '../../u8a'
+
+export const PADDING_TAG = Uint8Array.from([72, 79, 80, 82]) // "HOPR"
+export const PADDING_TAG_LENGTH = 4
+
+export function addPadding(msg: Uint8Array) {
```

Comment:
can you document what these functions do?

Reply:
Solved, please recheck